.circleci/publish.sh

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env bash
+
+set -eu
+
+echo $GPG_KEY | base64 --decode | gpg --batch --import --passphrase "$GPG_PASSPHRASE"
+
+gpg --passphrase $GPG_PASSPHRASE --batch --yes -a -b LICENSE
+
+mill -i src.io.iohk.decco.publish \
+--sonatypeCreds "$OSS_USERNAME:$OSS_PASSWORD"  \
+--gpgPassphrase "$GPG_PASSPHRASE" \
```

Comment:
As this is a public repository, what prevents people from submitting a PR containing code like `println(java.lang.System.getenv("GPG_PASSPHRASE"))` and looking into the logs to see the printed key?

Reply:
will they have access to circle ci

Reply:
Marcin has already addressed this concern in his PR.